HW8_Airport/Air_Simulation/Baggage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Air_Simulation
+{
+    class Baggage
+    {
+        public int weight;
```

Comment:
as we know, the best practice to use just private/protected fields. If we need to use public modifier - just create a  C# Property for that.